code/game/turfs/flooring/flooring_shuttle.dm

```diff
@@ -3,6 +3,10 @@
 	icon = 'icons/turf/shuttle.dmi'
 	abstract_type = /turf/simulated/floor/shuttle
 
+/turf/simulated/floor/shuttle/plating
+	icon_state = "plating"
```

Comment:
Why is this necessary? Does `/turf/simulated/floor` not work? Nothing does `istype(foo, /turf/simulated/floor/shuttle)` and there's no logic defined on this either.